hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/exec/systemcontracts/hts/create/ClassicCreatesCall.java

```diff
@@ -185,22 +186,34 @@ private ResponseCodeEnum validityOfSynthOp() {
         return OK;
     }
 
-    private VerificationStrategy verificationStrategyFor(@NonNull final MessageFrame frame) {
+    private VerificationStrategy verificationStrategyFor(
+            @NonNull final MessageFrame frame, @NonNull final TokenCreateTransactionBody op) {
         final var legacyActivation = legacyActivationIn(frame);
 
-        // Choose a dispatch verification strategy based on whether the legacy
-        // activation address is active (somewhere on the stack)
+        // If there is a crypto admin key, we need an either/or strategy to let it
```

Comment:
I would have thought this change, though to a `private` method, was significant enough to cause some need to modify an existing test over at `ClassicCreatesCallTest` or add a test there - but that file isn't in this PR.

Reply:
Yes `CreatesCall` needs to be made more testable...will punt on this one for now. 😞